Terminal.Gui/Views/FrameView.cs

```diff
@@ -85,9 +86,10 @@ class ContentView : View {
 		/// <param name="title">Title.</param>
 		/// <param name="views">Views.</param>
 		/// <param name="border">The <see cref="Border"/>.</param>
-		public FrameView (Rect frame, ustring title = null, View [] views = null, Border border = null) : base (frame)
+		public FrameView (Rect frame, ustring title = null, View [] views = null, Border border = null) //: base (frame)
```

Comment:
`Terminal.Gui.ViewTests.FrameViewTests.Constuctors_Defaults` is failing.

If I remove the comment so it is calling the base constructor again it passes.  I think you made this change @tig, is there a reason not to call base? and if so does it mean the test needs changed? Or should I just remove the comment?